### PR TITLE
MC-15297: doc for retainEnvironment on tabs

### PR DESCRIPTION
**Pull request by AnKhangC**
#### Changes made
Add retainEnvironment boolean.

#### Related PRs
- []()

### CODE REVIEW

File: source/includes/administration/_custom_navigation.md [hunk 1]
```diff
@@ -124,6 +124,8 @@ Attributes | &nbsp;
 `tabs.key`<br/>*String* | The view key for a tab of type `SYSTEM`. 
 `tabs.skipEnvironmentList`<br/>*boolean* | Skip the environment list if there is a single environment associated with the service tab. Configurable for a tab of type `SERVICE`.
 `tabs.showSubTabs`<br/>*boolean* | Whether or not to show the subtabs, as defined by the plugin, underneath the menu tab. Configurable for a tab of type `SERVICE`.
+`tabs.retainEnvironment`<br/>*boolean* | Skip the environment picker and retain the environment context when navigating between tabs with the same service connection. Configurable for a tab of type `SERVICE`.
```
Comment (patrickspensieri) [inline source/includes/administration/_custom_navigation.md]: What do you guys think about
> Retain the selected environment when navigating between tabs with the same service connection. Configurable for a tab of type `SERVICE`.

Comment (AnKhangC) [reply source/includes/administration/_custom_navigation.md]: Sounds good will make the changes

Comment (AnKhangC) [reply source/includes/administration/_custom_navigation.md]: efbcdc1

**Review by patrickspensieri — APPROVED** [review]
👍

**Review by HassanHaidar — APPROVED** [review]

File: source/includes/administration/_custom_navigation.md [hunk 2]
```diff
@@ -299,6 +302,8 @@ Optional | &nbsp;
 `tabs.key`<br/>*String* |  *Required* if type `SYSTEM`. The view key for a tab of type `SYSTEM`. 
 `tabs.skipEnvironmentList`<br/>*boolean* | Skip the environment list if there is a single environment associated with the service tab. Configurable for a tab of type `SERVICE`.
 `tabs.showSubTabs`<br/>*boolean* | Whether or not to show the subtabs, as defined by the plugin, underneath the menu tab. Configurable for a tab of type `SERVICE`.
+`tabs.retainEnvironment`<br/>*boolean* | Retain the selected environment when navigating between tabs with the same service connection. Configurable for a tab of type `SERVICE`.
```
Comment (m-orfanos) [inline source/includes/administration/_custom_navigation.md:305]:
```suggestion
`tabs.retainEnvironment`<br/>*Boolean* | Retain the selected environment when navigating between tabs with the same service connection. Configurable for a tab of type `SERVICE`. 
```

File: source/includes/administration/_custom_navigation.md [hunk 3]
```diff
@@ -299,6 +302,8 @@ Optional | &nbsp;
 `tabs.key`<br/>*String* |  *Required* if type `SYSTEM`. The view key for a tab of type `SYSTEM`. 
 `tabs.skipEnvironmentList`<br/>*boolean* | Skip the environment list if there is a single environment associated with the service tab. Configurable for a tab of type `SERVICE`.
 `tabs.showSubTabs`<br/>*boolean* | Whether or not to show the subtabs, as defined by the plugin, underneath the menu tab. Configurable for a tab of type `SERVICE`.
+`tabs.retainEnvironment`<br/>*Boolean* | Retain the selected environment when navigating between tabs with the same service connection. Configurable for a tab of type `SERVICE`. 
+
```
Comment (m-orfanos) [inline source/includes/administration/_custom_navigation.md]:
```suggestion
`tabs.retainEnvironment`<br/>*boolean* | Retain the selected environment when navigating between tabs with the same service connection. Configurable for a tab of type `SERVICE`. 

```

Comment (AnKhangC) [reply source/includes/administration/_custom_navigation.md]: Thanks for the changes

Comment (AnKhangC) [reply source/includes/administration/_custom_navigation.md]: @m-orfanos How did you make the build pass?

**Pull request merged**